src/components/DefaultImage.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function DefaultImage({idx, onClick, src, srcSet, sizes, height, width, alt, margin}){
+	style.margin = margin;
+	return (
+		<div style={style}> 
+			<a href="#" onClick={(e) => onClick(idx, e)}>
```

Comment:
I would switch the order of arguments here. As the `event` is always the first argument in event handlers. It's more convenient to add the extra data as second.

Also; wouldn't it be more convenient for the end user if he had more data? Something like: `onClick(e, { idx, image });`. Where image is the original image object, past to gallery. `photos[idx]`.